osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -58,39 +58,41 @@ func addCreateOsConfigTest(pkgTestSetup []*packageManagementTestSetup, testProje
 	packageName := "cowsay"
 	for _, tuple := range platformPkgManagers {
 		var oc *osconfigpb.OsConfig
-		var image string
 		uniqueSuffix := utils.RandString(5)
 
 		switch tuple.platform {
 		case "debian":
-			image = debianImage
-			pkgs := []*osconfigpb.Package{osconfigserver.BuildPackage(packageName)}
-			oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s-%s", path.Base(image), testName, uniqueSuffix), desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			for _, image := range debianImages {
+				pkgs := []*osconfigpb.Package{osconfigserver.BuildPackage(packageName)}
+				oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s-%s", path.Base(image), testName, uniqueSuffix), desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
```

Comment:
minor: consider breaking out the `fmt.Sprintf()` and `BuildAptPackageConfig()` to separate variables. It'll make the line shorter and the code easier to read.

Reply:
done